src/view/boot.ts

```diff
@@ -78,6 +77,7 @@ new AppConfigurationService().getConfigProperties().subscribe(config => {
         MdIconRegistry,
         provide('dotcmsConfig', {useValue: config.dotcmsConfig}),
         provide('routes', {useValue: routes}),
+        provide('user', {useValue: config.dotcmsConfig.configParams.user}),
```

Comment:
Right now this user is storage in LoginService, we have to define how storage the current context.